compiler/tools/fpp-to-cpp/test/fprime/Fw/Types/Types.fpp

```diff
@@ -13,6 +13,12 @@ module Fw {
     NO_ROOM_LEFT  @< No room left in the buffer to serialize data
   }
 
+  struct SMSignals {
```

Comment:
Do we still need this type?

Reply:
I removed this type and fixed a related bug in the code gen (a use of the type was still hanging around and needed to be replaced).